quantecon/game_theory/tests/test_normal_form_game.py

```diff
@@ -540,25 +533,12 @@ def test_best_response_2p():
         [np.array([1, 0]), np.array([0.5, 0.5]), np.array([0, 1])],
         'brs_expected': [0, 1, 2]
     }
-    test_case1 = {
-        'payoff_array': np.zeros((2, 3)),
-        'mixed_actions': [np.array([1, 0, 0]), np.array([1/3, 1/3, 1/3])],
-        'brs_expected': [0, 0]
-    }
+    test_case1 = {'payoff_array': np.zeros((2, 3)), 'mixed_actions': [np.array(
+        [1, 0, 0]), np.array([1 / 3, 1 / 3, 1 / 3])], 'brs_expected': [0, 0]}
```

Comment:
This might look better?

```suggestion
    test_case1 = {
        'payoff_array': np.zeros((2, 3)),
        'mixed_actions':
        [np.array([1, 0, 0]), np.array([1 / 3, 1 / 3, 1 / 3])],
        'brs_expected': [0, 0]
    }
```